src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -11,139 +9,125 @@ import { modalInstructions } from '../../../utils/helpers';
-          modalVisible={modalVisible}
-          handleCancel={() => {
-            setModalVisible(false);
-          }}
-          handleOk={() => {
-            setModalVisible(false);
-          }}
-          instructions={modalInstructions.matchUp}
-        />
-      )} */}
-      {/* <div className="matchup-container">
-        <Row className="toprow">
-          <Col className="green-box" xs={24} sm={13}>
-            {faceoffs[0] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[0]}
-                backgroundColor={'#C9E952'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={7}
-                dayNeededToUnlock={5}
-                hourNeededToUnlock={18}
-              />
-            )}
-          </Col>
-          <Col className="red-box" xs={24} sm={11}>
-            {faceoffs[1] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[1]}
-                backgroundColor={'#e97451'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={7}
-              />
-            )}
-          </Col>
-        </Row>
-        <Row className="bottomrow">
-          <Col className="yellow-box" xs={24} sm={11}>
-            {faceoffs[2] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[2]}
-                backgroundColor={'#ffde3b'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={8}
-              />
-            )}
-          </Col>
-          <Col className="blue-box" xs={24} sm={13}>
-            {faceoffs[3] && (
-              <FaceoffContent
-                custom_date={props.custom_date}
-                content={faceoffs[3]}
-                backgroundColor={'#438eac'}
-                votesRemaining={props.votesRemaining}
-                votesNeededToUnlock={9}
-              />
-            )}
-          </Col>
-        </Row>
-        <Button className="back-button" onClick={back2Dash}>
-          Back
-        </Button>
+      <div className="drawing-story-matchup">
+        <div className="small-matchup-window">
+          <div id="first-window">
+            <div className="player-container">
+              <div
+                id="first-player-small-window"
+                className="player-image-small"
+              ></div>
+              <figcaption>player name</figcaption>
+            </div>
+            <div className="vs-lightning-image"></div>
+            <div className="player-container">
+              <div
+                id="second-player-small-window"
+                className="player-image-small"
+              ></div>
+              <figcaption>player name</figcaption>
+            </div>
+          </div>
+          <h2>140 points to win</h2>
+        </div>
```

Comment:
I'm not seeing id's being used for logic. Are you using them for logic or for style? if for style only use className. If you are getting the element by id somewhere else perfect.. but otherwise id's should not be used for style.